configure.ac

```diff
@@ -37,14 +37,23 @@
 #
 
 AC_PREREQ([2.63])
-AC_INIT([pbspro], [18.1.0], [pbssupport@altair.com])
+# Use PBS_VERSION to override the version statically defined here. For example:
+# ./configure PBS_VERSION=19.1.2 --prefix=/opt/pbs
+AC_INIT([PBS Professional],
+  [19.0.0],
+  [pbssupport@altair.com],
+  [pbspro],
+  [http://www.pbspro.org/])
 AC_CONFIG_HEADERS([src/include/pbs_config.h])
 AC_CONFIG_SRCDIR([src/cmds/qmgr.c])
 AC_CONFIG_AUX_DIR([buildutils])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CANONICAL_TARGET([])
-# FIXME: The subdir-objects option will be enabled by default in the future
-AM_INIT_AUTOMAKE([-Wall foreign])
+os_id=`grep ^ID= /etc/os-release | sed -n 's/.*"\(.*\)"/\1/p'`
```

Comment:
Just curious to know: Will this cause problem on machine where /etc/os-release file is not present like RHEL 6?

Reply:
It shouldn't cause a problem if os_id ends up as an empty string. The AS_CASE below has a wildcard entry it will match. This bit of tooling allows us to deal with other exceptions we may encounter in the future by simply adding a new case strings to match.

Reply:
I also thought that but I was just confirming that. Thanks for confirming!